### PR TITLE
Questionnaire

### PR DESCRIPTION
**Pull request by Contrast-AlexB**
A quick once through for finding packages based on questionnaire.  Will continue to update in future to include misc (bluebird, and requests, etc.) + update readme.

Just wanted to get some initial feedback!

### CODE REVIEW

**Comment by Contrast-AlexB** [comment]
@cixel  - not sure how to attach reviewer.

File: nvnr.js [hunk 1]
```diff
@@ -54,80 +55,484 @@ function printSystemInfo () {
...
+
+
 // misc: request.js, bluebird, etc
-function misc (pkg) {
-	// sectionHeader('Other:');
+function printMiscInfo(dependencies, devDependencies) {
+	const miscInfo = new Section('Misc Used');
 
-	// console.log();
+	const asyncRegExp = new RegExp('\"async\"', 'i');
```
Comment (Contrast-AlexB) [inline nvnr.js]: I used https://www.npmjs.com/browse/depended to populate this

**Comment by Contrast-AlexB** [comment]
Awesome, also for the versions - are you not seeing them? Looks like this for me:

https://imgur.com/a/IcBik

**Comment by cixel** [comment]
Missed that. Looks great.

**Pull request merged**